components/ProductDetails/ProductDetailsOptionalTours.vue

```diff
@@ -113,12 +112,15 @@ export default {
         },
         markButtonActive: function(content) {
             EventBus.$emit('markButtonActive', content)
+        },
+        closeModal: function(event) {
+            console.log(event.target.parent)
```

Comment:
Do we need to do anything to closeModal?

Reply:
No, this is redundant code, I used the close button provided by Bootstrap-Vue. I will remove the close function.